ts/Series/OHLC/OHLCSeries.ts

```diff
@@ -394,7 +350,7 @@ interface OHLCSeries {
     pointAttrToOptions: Record<string, string>;
     pointClass: typeof OHLCPoint;
     init(): void;
-    toYData(point: OHLCPoint): OHLCPoint.PointShortOptions;
+    toYData(point: OHLCPoint): Array<number>;
 }
 extend(OHLCSeries.prototype, {
```

Comment:
Are all of the extended options (below) necessary here? For example `pointValKey`, `pointAttrToOptions` etc are inherited from `HLC` series, right?

Reply:
It's not resolved @pawellysy ;)